.travis.yml

```diff
@@ -7,7 +7,7 @@ matrix:
     - jdk: openjdk10
```

Comment:
You will need to drop the lines:
```
- jdk: openjdk8
- jdk: openjdk9
- jdk: openjdk10
```

You could also add `dist: bionic`.

Reply:
hi Adam, 
thanks again for your comments.
just discussed it with David and Laura, David wants us to keep supporting jdk8 for other products embedding droid. So I'll have to keep those.
just tried making new changes, will clean history if it goes through and replace
```
    - os: osx
      osx_image: xcode10.1
      env: JAVA_HOME=$(/usr/libexec/java_home)
```
with 
```
    - os: linux
      dist: bionic
```

Reply:
ah you probably want to keep the osx build, ok got it, will just add it on top

Reply:
So I don't understand what's happening then. You want to keep JDK 8 support, but you are moving to Java language level 11. My experience is that even if you back-compile, these things are not 100% compatible. Have I missed something?

Reply:
@adamretter shall we keep osx in the list ? or we can remove it and only use bionic?

Reply:
in previous DROID releases we've attempted to maintain backwards compatibility for Java where possible, as some users and particular instances that bundle DROID (preservation systems, toolkits etc) may have limitations on what they can install and may be tied back to earlier Java versions. /if possible/ I'd like to ensure 6.5 can run on 1.8, but need to understand whether this is possible or not.

Reply:
yeah right :)). Well I think there was some confusion regarding the task "shifting support forward to Java 11", which was already done by Radek. I guess the whole idea from David was more to update all dependencies.
I moved back to Java language level 8.
I'll rename the PR too

Reply:
@Dclipsham that will work fine on java 8!

Reply:
as discussed, we won't remove support to java 8. Will resolve this conversation